frontend/src/reducers/LeafletMouseSlice.tsx

```diff
@@ -0,0 +1,22 @@
+import { LeafletMouseEvent } from 'leaflet';
+import { createSlice, createAction } from '@reduxjs/toolkit';
+
+export const saveClickLatLng = createAction<LeafletMouseEvent>('saveClickLatLng');
+export const clearClickLatLng = createAction('clearLatLng');
+const leafletMouseSlice = createSlice({
```

Comment:
Looking forward to hear your experience with redux-toolkit...
Could you please add some JSDoc here as it is a different approach to app state management? Thanks!